src/requests/getForecast.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-console */
+import axios from "axios";
+
+const getForecast = (
+  searchText,
+  setSelectedDate,
+  setForecasts,
+  setLocation,
+  setErrorMessage
+) => {
+  let endpoint = "https://mcr-codes-weather-app.herokuapp.com/forecast";
+
+  if (searchText) {
+    endpoint += `?city=${searchText}`;
+  }
```

Comment:
Currently if the user clicks the search button without `searchText` you'll still make an API call so you'd also want this code nested in the if block